app/api/v1/chat.py

```
@@ -118,11 +141,14 @@
 @router.post("/voice/text")
 async def chat_voice_to_text(
     agent_id: str = Form(...),
+    channel: str = Form(...),
+    force_generate: bool = Form(...),
     image_file: Optional[UploadFile] = None,
     audio_file: UploadFile = File(...),
     api_key: str = Depends(get_api_key),
     llm_service: LLMService = Depends(get_llm_service),
     stt_service: STTService = Depends(get_stt_service),
+    schedule_service: ScheduleService = Depends(get_schedule_service),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)